### DISCUSSION TITLE
Help please!!! PnP - Search Results - users are seeing different results

### DISCUSSION THREAD

**Discussion by egwrk** [microsoft-search/pnp-modern-search#3780] [Q&A]
The pnp results are showing learning sessions that users can register for and some sessions are not showing for some people. None of the items are permissioned, so everyone should be seeing the same sessions. 

Two users in the US cannot see the in-person sessions on July 18. One user had filtered on Country = US and Virtual/In person = In person, hitting apply each time. The other US user is having the same issue. However, I and a few other users can see these sessions. There were changes made to the list recently so I'm assuming it is still reindexing. Does the PnP Search Results sync with the SharePoint list at different times for different users? 

Based on my understanding, it can take up to 24 hours for PnP Search Results to sync with the SharePoint list when the list gets modified. I've let them know about this and we will be checking the list tomorrow but was curious to know while the list is reindexing, if users will see different results until the reindex is completed. 

The stakeholders will review the page tomorrow (June 5) and we have communications going out this Thursday (June 6) 0900 ET linking to the page. I would greatly appreciate any assistance!

![image](https://github.com/microsoft-search/pnp-modern-search/assets/161059375/3a81e5d2-0430-4e8e-806b-18fed0de5650)

**Comment by wobba** [comment]
Assuming access control is the same for all users and the query is the same (I recommend sort on a property and not relevance), then they should see the same results. Sorting on relevance can show results in a different order, but should still be the same. And this has nothing to do with PnP Search Results web part, but with what the search API return. And and update to an item can indeed take time to be picked up for reindex - and no system SLA from Microsoft.
